legal-api/migrations/versions/da43f7bef692_.py

```diff
@@ -0,0 +1,43 @@
+"""empty message
+
+Revision ID: da43f7bef692
+Revises: d9e21a1b82ca
+Create Date: 2021-12-09 15:15:34.439924
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'da43f7bef692'
+down_revision = 'd9e21a1b82ca'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    state = postgresql.ENUM('ACTIVE', 'HISTORICAL', 'LIQUIDATION', name='state')
+    state.create(op.get_bind(), checkfirst=True)
+    op.add_column('businesses', sa.Column('state', state, nullable=True))
```

Comment:
what happens to the existing data when we run the migration for this column(`businesses.state`)?  is it null?

Reply:
We usually keep the DML out of migration. We will have to run a script to update the state of existing businesses as ACTIVE.